k9mail/src/test/java/com/fsck/k9/helper/EmailHelperTest.java

```diff
@@ -0,0 +1,30 @@
+package com.fsck.k9.helper;
+
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class EmailHelperTest {
+
+    @Test
+    public void getDomainFromEmailAddress_returnsDomainForValidEmail() {
```

Comment:
The name of the method already says what should be returned. In such cases I think it's best to only describe the input. In this case maybe `getDomainFromEmailAddress_withRegularEmailAddress`.